share/spice/thumbtack/thumbtack.js

```diff
@@ -0,0 +1,47 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_thumbtack = function(api_result){
+
+        // Don't show anything if we weren't able to return 2 or more services.
+        // In future iterations, we should support a different view in the case
```

Comment:
Ah yes. We actually don't return single-element results on our end anyway, so I wasn't able to test it. I can remove it.